src/api/request.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+
+export default async (url, method, data = {}) => {
+  try {
+    const response = await axios({
+      method,
+      url,
+
+      data: { ...data },
+    });
+    return Promise.resolve(response.data);
+  } catch (error) {
+    return Promise.reject(error.response.data.message);
+  }
+};
+
+export const request = async (
```

Comment:
The request function you added is cool as it will help reduce the lines of code written for actions. It would be nice if you sync with teammates who used the one above so that their codes can be refactored to use the second one. This is to avoid having two functions that serve the same purpose.

Reply:
Thanks for bringing this to knowledge. I will sure do good to let the team know of the update